doc/nrf/shortcuts.txt

```diff
@@ -23,7 +23,7 @@
 .. |nRF51DK| replace:: nRF51 DK board (PCA10028)
 .. |nRF52DK| replace:: nRF52 DK board (PCA10040)
 .. |nRF52840Dongle| replace:: nRF52840 Dongle (PCA10059)
-.. |Thingy91| replace:: Thingy:91 (PCA20035) - see :ref:`ug_thingy91`
+.. |Thingy91| replace:: Thingy:91 (THINGY91) - see :ref:`ug_thingy91`
```

Comment:
This should stay as is - the PCA number is still correct, and here we are not referring to the build target.